### PR TITLE
 * Creating validations of Contact

### PR DESCRIPTION
**Pull request by GabrielDeveloper**

### CODE REVIEW

File: app/Http/Requests/ContactRequest.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'first_name' => 'required|max:20',
```
Comment (Cayan) [inline app/Http/Requests/ContactRequest.php]: can you add the string type to this validation?

File: app/Http/Requests/ContactRequest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            'last_name' => '',
```
Comment (Cayan) [inline app/Http/Requests/ContactRequest.php]: can you add the string type to this validation?

**Review by Cayan — CHANGES_REQUESTED** [review]

File: app/Http/Requests/ContactRequest.php (hunk 1 above)
Comment (GabrielDeveloper) [reply app/Http/Requests/ContactRequest.php]: It's done! =)

File: app/Http/Requests/ContactRequest.php (hunk 2 above)
Comment (GabrielDeveloper) [reply app/Http/Requests/ContactRequest.php]: It's done! =)

**Pull request merged**